test/test.js

```diff
@@ -4770,6 +4770,85 @@ exports['parallel consume from async generator'] = function (test) {
     });
 };
 
+exports['parallel - behaviour of parallel with fork() - issue #234'] = function (test) {
+    test.expect(1);
+
+    function addTen(a) {
+        return a + 10;
+    }
+
+    function addTwenty(a) {
+        return a + 20;
+    }
+
+    var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+    var expected = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9,
+        10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
+        20, 21, 22, 23, 24, 25, 26, 27, 28, 29 ];
+
+    var baseRange = _(arr);
+    var loRange = baseRange.fork();
+    var midRange = baseRange.fork().map(addTen);
+    var hiRange = baseRange.fork().map(addTwenty);
+
+    _([loRange, midRange, hiRange])
+        .parallel(3)
+        .toArray(function (xs) {
+            test.same(xs, expected);
+            test.done();
+        });
+};
+
+exports['parallel - behaviour of parallel with fork() - invariant violation - issue #234'] = function  (test) {
```

Comment:
This isn't "behavior of parallel with fork()". It's more "parallel consumption liveness". Also, reference this issue (302) instead of 234.